src/jquery.selectric.js

```diff
@@ -555,52 +565,56 @@
       });
 
       // Toggle open/close
-      _this.elements.wrapper.on('click' + bindSufix, function(e) {
+      _this.elements.wrapper.on('click' + eventNamespaceSuffix, function(e) {
         _this.state.opened ? _this.close() : _this.open(e);
       });
 
-      // Translate original element focus event to dummy input
-      _this.$element.on('focus' + bindSufix, function() {
-        _this.elements.input.focus();
-      });
-
-      _this.elements.input
-        .prop({ tabindex: _this.originalTabindex, disabled: false })
-        .on('keydown' + bindSufix, $.proxy(_this.handleKeys, _this))
-        .on('focusin' + bindSufix, function(e) {
-          _this.elements.outerWrapper.addClass(_this.classes.focus);
+      // Translate original element focus event to dummy input.
+      // Disabled on mobile devices because the default option list isn't
+      // shown due the fact that hidden input gets focused
+      if ( !(_this.options.nativeOnMobile && _this.utils.isMobile()) ) {
```

Comment:
This was needed because the hidden input (which are used for key based search) steals the focus and the option list was not opened. Also the key based search is not needed on mobile devices...